tools/generator/Tests-Core/expected.ji/Android.Text.SpanTypes.cs

```diff
@@ -1,5 +1,6 @@
 namespace Android.Text {
   public enum SpanTypes {
+    [global::Android.Runtime.IntDefinition (null, JniField = "I:android/text/Spanned.SPAN_COMPOSING")]
```

Comment:
I'm still not clear on why some of these values have a type prefix -- `I:` -- and some don't.

Perhaps the type should be separated from the member name, for simplicity?

    [IntDefinition (JniSourceFieldType="I", JniSourceDeclaringType="android/text/Spanned", JniSourceFieldMember="SPAN_COMPOSING")]

Perhaps the type+member can be combined with a `.` separator, but I'm not sure we should have `<jni-type>:` as an *optional* prefix.